frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -293,6 +277,25 @@ const determineAvailableFilters = (initialFilters, items, filterGroups) => {
     );
   });
 
+  // Always show both install state filters
+  if (!filters.installState) {
```

Comment:
Can you explain the need to change the logic here? Previously, we always set the `installState` filters without this check. I'm assuming it's overwriting the value for `active`, but I'm not sure what's changed in the PR to cause that.

Reply:
Previously we set `filters.installState` at the top of this function, then iterated over each filter group passed in to initialize the filters with the possible values. I attempted to change the `Installed` and `Not Installed` labels in this object, but found that the change was not taking effect in the UI.

Turns out we pass `installState` in as a filter group (though I'm assuming this logic was added as that is not always the case). This means that the logic [here](https://github.com/openshift/console/pull/9450/files#diff-f5c2166645f6d9f295216d0a8e637a7e76d633490cbf3a4b94ed3aed11c5d27dR264) and [here](https://github.com/openshift/console/pull/9450/files#diff-f5c2166645f6d9f295216d0a8e637a7e76d633490cbf3a4b94ed3aed11c5d27dR276) was overwriting the initial `filters.installState` we set at the top of the function.

It was simpler/safer IMO to move the check below the loop. I can simplify this logic if `Installed` and `Not Installed` are the only two possible values for `installState`.